src/screens/Admin/screens/InstructorPulse/index.js

```diff
@@ -10,13 +12,15 @@ export default ({instructors}) => (
     main={
       <InstructorsList
         title={unpublishedTitleText}
-        instructors={filter(instructors, ['published_lessons', 0])}
+        instructors={filter(instructors, ['published_lessons', 0])
+          .sort(sortBy('-submitted_lessons', '-claimed_lessons', '-approved_lessons', '-id'))}
```

Comment:
They are actually sorted by ID which correlates to join date fairly nicely. I think that "currently waiting for approval" takes priority over "hasn't submitted" which is why I did it this way. Ultimately I'd like live filters that I could configure.